src/main/java/VASSAL/build/module/ModuleExtension.java

```diff
@@ -122,30 +122,24 @@ public void build() {
     // Record that we are currently building this Extension
     GameModule.getGameModule().setGpIdSupport(this);
 
-    BufferedInputStream in = null;
-    try {
-      in = new BufferedInputStream(archive.getInputStream(fileName));
-    }
-// FIXME: should this be a FileNotFoundException?
-    catch (IOException e) {
-    }
-
-    if (in == null) {
-      build(null);
-    }
-    else {
```

Comment:
This doesn't preserve behavior in the `in == null` case.

Reply:
The `in==null` case could only happen if `archive.getInputStream(fileName)` in pre-change line 127 threw an exception. This case is preserved and has better error messages now, it jumps from the outer `try` into either the `catch (FileNotFoundException e)` in line 138 or the `catch (IOException e) {` in line 141.

The `build(null)` did nothing, the `build(Element e)` method starts like this:
```
  public void build(Element e) {
    if (e == null) {
      return;
    }
```